### PR TITLE
DM-28347: Add definitions

### PR DESCRIPTION
**Pull request by mfisherlevine**

### CODE REVIEW

File: etc/glossarydefs.csv [hunk 1]
```diff
@@ -853,6 +854,7 @@ Apache Parquet,A columnar storage data persistence format maintained by the Apac
...
 Archive,"The repository for documents required by the NSF to be kept. These include documents related to design and development, construction, integration, test, and operations of the LSST observatory system. The archive is maintained using the enterprise content management system DocuShare, which is accessible through a link on the project website www.project.lsst.org",Adm,,,G
 Archive Center,"Part of the LSST Data Management System, the LSST archive center is a data center at NCSA that hosts the LSST Archive, which includes released science data and metadata, observatory and engineering data, and supporting software such as the LSST Software Stack",DM,,,G
+Archiver,The IIP component responsible for transferring raw images and metadata to OODS and DBB in real time,TS,,Catch-up Archiver,G
```
Comment (timj) [inline etc/glossarydefs.csv:857]: Isn't the archiver a DM thing

File: etc/glossarydefs.csv [hunk 2]
```diff
@@ -891,6 +893,7 @@ Camcol,"In the SDSS survey, a camera column is the range (in declination) covere
 Camera,"The LSST subsystem responsible for the 3.2-gigapixel LSST camera, which will take more than 800 panoramic images of the sky every night. SLAC leads a consortium of Department of Energy laboratories to design and build the camera sensors, optics, electronics, cryostat, filters and filter exchange mechanism, and camera control system",CAM,,,G
 camera,"An imaging device mounted at a telescope focal plane, composed of optics, a shutter, a set of filters, and one or more sensors arranged in a focal plane array",Sci,,,G
 Camera Crosstalk-Corrected Image,An image from the Camera system that has had crosstalk removed but has not been processed by the Instrument Signature Removal pipeline,DM,,"C3 Image, Raw Image",G
+Catch-up Archiver,The Archiver for any images missed by the real time archiver,TS,,Archiver,G
```
Comment (timj) [inline etc/glossarydefs.csv:896]: Also DM isn't it?

**Review by timj — APPROVED** [review]
Looks fine to me apart from archiver being T&S vs DM

File: etc/glossarydefs.csv (hunk 1 above)
Comment (mfisherlevine) [reply etc/glossarydefs.csv:857]: I actually don't know exactly where the boundaries are drawn tbh. @ktlim?

Comment (ktlim) [reply etc/glossarydefs.csv:857]: Yes, it's a DM thing.

File: etc/glossarydefs.csv [hunk 3]
```diff
@@ -1036,6 +1039,7 @@ NCSA Facility,"The data center at the National Center for Supercomputing Applica
 Nightly Alert Processing,Deprecated term; see 'Alert Production',DM,,Alert Production,G
 Nightly Archive Processing,Deprecated term; see 'Prompt Processing',DM,,Prompt Processing,G
 Non-Standard Visit,"Any single observation of a LSST field that is not comprised of either two 15 second 'Snap' exposures (a standard visit) or one 30 second exposure (an alternative standard visit). For example, exposure times for Special Programs might be significantly shorter or longer than a standard visit (or of random length)",DM,DMTN-065,,G
+nublado,The service underpinning the Notebook Aspect part of the Rubin Science Platform,DM,,G
```
Comment (ktlim) [inline etc/glossarydefs.csv:1042]: "part" is kind of redundant with Aspect, no?

Comment (mfisherlevine) [reply etc/glossarydefs.csv:1042]: Ha! I spontaneously said exactly that when talking to Frossieyesterday, but my thoughts were, on writing that, that "Notebook Aspect" is a phrase, and it scans very weirdly without the somewhat-redundant "part". Would "component" scan better for you? I'm happy to change it as I had exactly the same thought, but liked it much less without nothing there at all...

Comment (ktlim) [reply etc/glossarydefs.csv:1042]: I'm not sure what you mean by "a phrase".  The use of "Notebook Aspect of the Rubin Science Platform" is official: see the title of SQR-025, the text of LSE-319, etc.

Comment (mfisherlevine) [reply etc/glossarydefs.csv:1042]: Oh, I know it's official, that was kind of the point! By "phrase" what I really meant was "compound noun" and thus, when referring to the Notebook-Aspect component of the system, the secondary noun (be it component, part, or something else) was still warranted, but perhaps I'm just reading it wrong.

Comment (ktlim) [reply etc/glossarydefs.csv:1042]: It's like "the Legislative Branch of the US Government".  "Notebook" is the qualifier, "Aspect" is the part-noun.

File: etc/glossarydefs.csv [hunk 4]
```diff
@@ -1039,6 +1042,7 @@ NCSA Facility,"The data center at the National Center for Supercomputing Applica
 Nightly Alert Processing,Deprecated term; see 'Alert Production',DM,,Alert Production,G
 Nightly Archive Processing,Deprecated term; see 'Prompt Processing',DM,,Prompt Processing,G
 Non-Standard Visit,"Any single observation of a LSST field that is not comprised of either two 15 second 'Snap' exposures (a standard visit) or one 30 second exposure (an alternative standard visit). For example, exposure times for Special Programs might be significantly shorter or longer than a standard visit (or of random length)",DM,DMTN-065,,G
+nublado,The service underpinning the Notebook Aspect of the Rubin Science Platform,DM,,G
```
Comment (ktlim) [inline etc/glossarydefs.csv:1045]: This is missing a comma.  Please fix ASAP.

File: etc/glossarydefs.csv [hunk 5]
```diff
@@ -1102,6 +1106,7 @@ Risk Management,"The art and science of planning, assessing, and handling future
 Safety,The control of accidental loss,Adm,,,G
 Safety Council,A consulting body providing policy advice and evaluation of safety program effectiveness; the council is composed of independent safety professionals and representatives of LSST institutional members,Adm,,,G
 Safety Manager,"The person who manages, executes, and verifies compliance with the LSST Safety Policy (LPM-18); the Safety Manager is also chair of the Safety Council",Adm,,,G
+SAL script,A program which communicates via SAL messages and adheres to a specific API, performing coordinated telescope and instrument control operations, such as "slew to a target and take an image", or "take a series of flats",TS,,script queue,G
```
Comment (ktlim) [inline etc/glossarydefs.csv:1109]: Comma-containing definitions need to be quoted.  Please fix ASAP.

Comment (mfisherlevine) [reply etc/glossarydefs.csv:1109]: DM-28517 filed, sorry 😔

File: etc/glossarydefs.csv (hunk 4 above)
Comment (mfisherlevine) [reply etc/glossarydefs.csv:1045]: DM-28517 filed, sorry 😔

**Pull request merged**